amo2kinto/tests/test_importer.py

```diff
@@ -274,6 +280,37 @@ def test_sync_records_calls_the_scenario():
     RECORDS = json.load(f)
 
 
+def test_sync_records_validate_records_against_schema():
+    with mock.patch(
+            'amo2kinto.importer.get_kinto_records',
+            return_value=[]):
+        with mock.patch('amo2kinto.importer.push_changes'):
+
+            sync_records(RECORDS['addons'],
+                         FIELDS['addons'],
+                         mock.sentinel.kinto_client,
+                         mock.sentinel.bucket,
+                         mock.sentinel.collection,
+                         SCHEMAS['addons'],
+                         mock.sentinel.permissions)
```

Comment:
I can add a try catch to make it expecit that it shouldn't raise a ValidationError.